spring-security-mvc-socket/pom.xml

```diff
@@ -11,9 +11,9 @@
 
     <parent>
         <groupId>com.baeldung</groupId>
-        <artifactId>parent-spring-5</artifactId>
+        <artifactId>parent-spring-4</artifactId>
```

Comment:
Are we downgrading from Spring 5.x to 4.x?

Reply:
@KevinGilmore - just saw your comment on here. I replied over on JIRA - there was an unintentional upgrading from the original dependencies used by the other WebSocket article in here. It should have stayed at Spring-4. Spring-5 encourages SSE and Reactive WebSockets. This article specifies how to improve Spring WebSockets to send messages to specific users and not broadcast them to everyone. Sorry for any confusion!